code/game/objects/items/theft_tools.dm

```diff
@@ -14,6 +14,7 @@
 	var/pulse = 0
 	var/cooldown = 0
 	var/pulseicon = "plutonium_core_pulse"
+	var/list/blacklisted_locs = list(/obj/machinery/nuclearbomb,  /obj/item/nuke_core_container)
```

Comment:
Rename this to `var/static/list/blacklisted_locs_typecache = typecacheof(list(/obj/machinery/nuclearbomb,  /obj/item/nuke_core_container))`